pkg/image/registry.go

```diff
@@ -14,9 +14,9 @@ type Registry interface {
 	// If the referenced image does not exist in the registry, an error is returned.
 	// Push(ctx context.Context, ref string) error
 
-	// Unpack writes the unpackaged content of an image to a directory.
+	// Unpack recursively copies the filesystem contents of an image from the specified location to a local directory.
 	// If the referenced image does not exist in the registry, an error is returned.
-	Unpack(ctx context.Context, ref Reference, dir string) error
+	Unpack(ctx context.Context, ref Reference, from, to string) error
```

Comment:
nit: I wonder if `to` should become `fs.FS`, or if that's too prescriptive.

Reply:
All we really need is a filepath to write to, but I can see if fs.FS is a good option if it brings some additional benefits.

Reply:
it's probably a discussion for a later date.